Modules/_posixsubprocess.c

```diff
@@ -1192,7 +1195,7 @@ subprocess_fork_exec_impl(PyObject *module, PyObject *process_args,
 #endif /* HAVE_SETREUID */
     }
 
-    c_fds_to_keep = PyMem_Malloc(fds_to_keep_len * sizeof(int));
+    c_fds_to_keep = PyMem_New(int, fds_to_keep_len);
```

Comment:
It never overflows. `fds_to_keep_len` is the size of a tuple. `sizeof(int) <= sizeof(PyObject *)`.